src/main/kotlin/com/moshbit/katerbase/MongoHelper.kt

```diff
@@ -406,3 +406,12 @@ class AggregationPipeline {
     }
   }
 }
+
+class QueryStats(
+  val executionPipeline: List<String>,
+  val executionSuccess: Boolean,
+  val returnedDocumentCount: Int,
```

Comment:
```suggestion
  val returnedDocuments: Int,
```
"count" or "number" is mostly not needed ;)